components/layout/app-footer.tsx

```diff
@@ -0,0 +1,82 @@
+import { GitHubLogoIcon, TwitterLogoIcon } from '@radix-ui/react-icons';
+import { Flex, Link, Text } from '@radix-ui/themes';
+import React from 'react';
+
+import { AppContent } from './app-content';
+
+// @TODO: replace with real links: SUPA-27
+const FOOTER_SERVICE_LINKS = [
+  {
+    href: 'https://experiment.st',
+    label: 'Privacy policy',
+  },
+  {
+    href: 'https://experiment.st',
+    label: 'Terms of service',
+  },
+] as const;
+
+const FOOTER_SOCIAL_LINKS = [
+  {
+    href: 'https://github.com/okanisildar/beecast',
+    icon: <GitHubLogoIcon />,
+    title: 'GitHub',
+  },
+  {
+    href: 'https://twitter.com/beecast_ai',
+    icon: <TwitterLogoIcon />,
+    title: 'Twitter',
+  },
+] as const;
+
+export function AppFooter() {
+  return (
+    <AppContent>
+      <Flex justify="between" py="4">
+        <Flex align="start" direction="column">
+          <Text size="2" weight="bold">
+            Built with ☕️ by your folks at the{' '}
+            <Link color="gray" href="https://experiment.st" target="_blank">
```

Comment:
https://linear.app/experiment-station/issue/SUPA-28/basic-homepage-for-experimentst